### PR TITLE
dev/reference/key-monitoring-metrics, TOC: rename dashboard files

### PR DESCRIPTION
**Pull request by CaitinChen**
@lilin90 PTAL

### CODE REVIEW

**Comment by morgo** [comment]
Please add aliases when renaming files (the old URLs could be stuck in bookmarks or search engine indexes). Otherwise LGTM

**Comment by CaitinChen** [comment]
@morgo Thanks~

File: dev/reference/key-monitoring-metrics/overview-dashboard.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Key Metrics
 summary: Learn some key metrics displayed on the Grafana Overview dashboard.
 category: reference
-aliases: ['/docs/op-guide/dashboard-overview-info/']
+aliases: ['/docs/op-guide/dashboard-overview-info/'],['/docs/dev/reference/key-monitoring-metrics/overview/']
```
Comment (kennytm) [inline dev/reference/key-monitoring-metrics/overview-dashboard.md]: I think the correct syntax should be 

```suggestion
aliases: ['/docs/op-guide/dashboard-overview-info/', '/docs/dev/reference/key-monitoring-metrics/overview/']
```

Comment (CaitinChen) [reply dev/reference/key-monitoring-metrics/overview-dashboard.md]: @kennytm Thanks~

**Comment by CaitinChen** [comment]
@lilin90 PTAL

**Pull request merged**